libs/gltfio/src/ResourceLoader.cpp

```diff
@@ -503,17 +503,28 @@ bool ResourceLoader::loadResources(FFilamentAsset* asset, bool async) {
             slot.vertexBuffer->setBufferObjectAt(engine, slot.bufferIndex, bo);
             continue;
         }
-        assert(slot.indexBuffer);
-        if (accessor->component_type == cgltf_component_type_r_8u) {
-            const size_t size16 = size * 2;
-            uint16_t* data16 = (uint16_t*) malloc(size16);
-            convertBytesToShorts(data16, data, size);
-            IndexBuffer::BufferDescriptor bd(data16, size16, FREE_CALLBACK);
+        else if (slot.indexBuffer) {
```

Comment:
nit: move the `else if` so it's on the same line as the `}` (or just delete the `else`)

Reply:
Thanks it's fixed. BTW what is nit?

Reply:
nit is short for [nitpick](https://www.google.com/search?q=nitpick&oq=nitpick&aqs=chrome..69i57j69i59l3j69i60l4.742j0j7&sourceid=chrome&ie=UTF-8)